hitl/streaming.py

```diff
@@ -17,8 +17,8 @@ async def _main():
                         help="The MQTT topic prefix of the target")
     parser.add_argument("--broker", "-b", default="mqtt", type=str,
                         help="The MQTT broker address")
-    parser.add_argument("--host", default="0.0.0.0",
-                        help="Local address to listen on")
+    parser.add_argument("--multicast", "-m", default="239.192.1.100", type=str,
+                        help="The multicast address to use for streaming")
```

Comment:
`type=str` is not necessary IIRC.
And let's keep the help string `IP address to listen on`.